vault/changelog.release.2021-11-16.md

```diff
@@ -0,0 +1,55 @@
+---
+id: NWuXh9ZreX3XKI6NhRIBJ
+title: "0.69"
+desc: ''
+updated: 1637082185375
+created: 1637081497904
+---
+
+Dendron 0.69 has sprouted  🌱
+
+Updates include enhancements to using Lookup, troubleshooting schema errors, and tweaks to [[Native Workspaces|dendron.topic.workspace.native]].
+
+If you haven't taken a look at what Native Workspaces entail, they are a way to have Dendron docs within an existing project (such as a docs directory of a Python project).
```

Comment:
Native workspaces are a new addition to Dendron. They let you use Dendron inside your existing projects by turning a subdirectory (eg. `docs`) into a Dendron (Native) workspace. Note that Dendron Native workspace is a   [[seed|dendron://dendron.dendron-site/tags.stage.seed]] stage feature and that the interface is still being actively worked on between weekly releases.